DateTime.js

```diff
@@ -133,10 +136,12 @@ var Datetime = createClass({
 			formats.time = locale.longDateFormat('LT');
 		}
 
-		formats.datetime = formats.date && formats.time ?
-			formats.date + ' ' + formats.time :
-			formats.date || formats.time
-		;
+                if ( !formats.datetime  || formats.datetime === true ) {
+                        formats.datetime = formats.date && formats.time ?
```

Comment:
Could this be a separate function? It looks very confusing. Or maybe pull out time and date into variables?

Reply:
Yeah, very confusing. I will extract this.